src/App.js

```diff
@@ -1,9 +1,11 @@
+import React from 'react';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
   return (
```

Comment:
- [ ] You have applied React class components very well but, it is meant for the `App` component too. Kindly refactor the `App` component to a React class based component. :pray: